pkg/setting/setting_unified_alerting.go

```diff
@@ -330,6 +332,8 @@ func (cfg *Cfg) ReadUnifiedAlertingSettings(iniFile *ini.File) error {
 		LokiTenantID:          stateHistory.Key("loki_tenant_id").MustString(""),
 		LokiBasicAuthUsername: stateHistory.Key("loki_basic_auth_username").MustString(""),
 		LokiBasicAuthPassword: stateHistory.Key("loki_basic_auth_password").MustString(""),
+		FanoutPrimary:         stateHistory.Key("fanout_primary").MustString(""),
+		FanoutSecondaries:     splitTrim(stateHistory.Key("fanout_secondaries").MustString(""), ","),
```

Comment:
Should we add these fields to default config along with descriptions, supported values and examples?

Reply:
We plan on doing so Soon:tm: - we were intentionally avoiding documentation for now so we can break the config fields if we really need. The lack of documentation was a proxy for a "feature flag"